### PR TITLE
Ga/another feature 2

### PR DESCRIPTION
**Pull request by gualterandre**

### CODE REVIEW

**Comment by gualterandre** [comment]
testing

**Comment by MelvinPeepers** [comment]
Test comment

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "build": "echo 'CACHED_COMMIT_REF: '$CACHED_COMMIT_REF && echo 'COMMIT_REF: '$COMMIT_REF && next build",
+    "build": "echo 'CACHED_COMMIT_REF: '$CACHED_COMMIT_REF && echo 'COMMIT_REF: '$COMMIT_REF && echo 'BRANCH: '$BRANCH && next build",
```
Comment (gualterandre) [inline package.json:7]: like it!

**Comment by gualterandre** [comment]
image


<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Chrome 105.0.0.0 on Mac OS 10.15.7
Viewport:  1863 x 944 @2x
Language:  en-GB
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-4--sunny-clafoutis-4bbf9a.netlify.app%2F&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=105.0&resolution=1863x944&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-4--sunny-clafoutis-4bbf9a.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=63468f3370f7140008049d7a&commentID=634690cf1420e552e91050c7&resolution=resolved)

**Pull request merged**